tests/lib/rules/no-access-state-in-setstate.js

```diff
@@ -19,11 +20,17 @@ const parserOptions = {
   }
 };
 
+const settings = {
+  react: {
+    createClass: 'createClass'
+  }
+};
```

Comment:
what does adding this here help? none of your new test cases use createClass.

Reply:
@ljharb 

None of the ones I added use it but a lot of tests in this rule use `createClass`. 

The default pragma is `createReactClass`, and since I added component detection to this rule I need to do one of two things: update all tests using `createClass`  to the default pragma `createReactClass` or what I did here: to set the default pragma to `createClass`.

Reply:
Thanks, makes perfect sense.